examples/vhs/table-detail-tea.tape

```diff
@@ -0,0 +1,20 @@
+# This is a vhs script. See https://github.com/charmbracelet/vhs for more info.
+# To run this script, install vhs and run `vhs ./examples/vhs/table-detail-tea.tape`
+Output "target/table-detail-tea.gif"
+Set Theme "Aardvark Blue"
+Set Width 1400
```

Comment:
Constrain this to max 1200 - otherwise GitHub readmes look bad when they scale the image

Reply:
Ok, I'll also commit changes for table.tape, where I copied it from.